react-backend/database/queries/getVisitsFilteredBy.js

```diff
@@ -1,27 +1,104 @@
 const dbConnection = require('../dbConnection');
 
-const query = orderBy =>
-  `SELECT users.id, users.sex, users.yearofbirth, activities.name, visits.date
-  FROM users
-  INNER JOIN visits ON users.id=visits.usersid
-  INNER JOIN activities ON visits.activitiesid = activities.id
-  WHERE activities.cb_id = $1
-  ORDER BY ${orderBy}`;
-
-const getVisitsFilteredBy = (cbId, orderBy) => {
-  const orderByQuery = {
-    yearofbirth: query('users.yearofbirth'),
-    sex: query('users.sex'),
-    activity: query('activities.name'),
-    date: query('visits.date'),
+const query = (filterBy, orderBy) =>
+  `Select users.id, users.sex, users.yearofbirth, activities.name, visits.date from users inner join visits on users.id=visits.usersid inner join activities on visits.activitiesid = activities.id where activities.cb_id = $1 ${filterBy} ${orderBy}`;
+
+const arrayMatch = (arr, search) => {
+  return arr.some(el => el === search);
+};
+
+const getFiltersQuery = filterBy => {
```

Comment:
I'm going to have at refactoring this function, but for the meantime let's merge and I'll make a separate PR for it.